deploy/cluster_role_binding.yaml

```diff
@@ -14,4 +15,21 @@ subjects:
 - kind: ServiceAccount
   name: default
   namespace: default
+---
```

Comment:
I think this should be it's own file. This file is called `cluster_role_binding.yaml`, so we should make a new one called `role_binding.yaml`, or rename the file

Reply:
We tried, but we didn't manage to make the operator-sdk taking the resources from both files

Reply:
That's too bad. I don't quite understand the limitation :/